assets/xml/objects/object_giant.xml

```diff
@@ -1,63 +1,90 @@
-        <Limb Name="object_giant_Standardlimb_0078CC" Type="Standard" Offset="0x78CC" />
-        <Limb Name="object_giant_Standardlimb_0078D8" Type="Standard" Offset="0x78D8" />
-        <Limb Name="object_giant_Standardlimb_0078E4" Type="Standard" Offset="0x78E4" />
-        <Limb Name="object_giant_Standardlimb_0078F0" Type="Standard" Offset="0x78F0" />
-        <Limb Name="object_giant_Standardlimb_0078FC" Type="Standard" Offset="0x78FC" />
-        <Limb Name="object_giant_Standardlimb_007908" Type="Standard" Offset="0x7908" />
-        <Limb Name="object_giant_Standardlimb_007914" Type="Standard" Offset="0x7914" />
-        <Limb Name="object_giant_Standardlimb_007920" Type="Standard" Offset="0x7920" />
-        <Limb Name="object_giant_Standardlimb_00792C" Type="Standard" Offset="0x792C" />
-        <Limb Name="object_giant_Standardlimb_007938" Type="Standard" Offset="0x7938" />
-        <Limb Name="object_giant_Standardlimb_007944" Type="Standard" Offset="0x7944" />
-        <Limb Name="object_giant_Standardlimb_007950" Type="Standard" Offset="0x7950" />
-        <Limb Name="object_giant_Standardlimb_00795C" Type="Standard" Offset="0x795C" />
-        <Limb Name="object_giant_Standardlimb_007968" Type="Standard" Offset="0x7968" />
-        <Skeleton Name="object_giant_Skel_0079B0" Type="Flex" LimbType="Standard" Offset="0x79B0" />
-        <Animation Name="object_giant_Anim_008394" Offset="0x8394" />
-        <Animation Name="object_giant_Anim_0096E4" Offset="0x96E4" />
-        <Animation Name="object_giant_Anim_00A1C4" Offset="0xA1C4" />
-        <Animation Name="object_giant_Anim_00ACA4" Offset="0xACA4" />
-        <Animation Name="object_giant_Anim_00B784" Offset="0xB784" />
-        <Animation Name="object_giant_Anim_00C5D4" Offset="0xC5D4" />
-        <Animation Name="object_giant_Anim_00D040" Offset="0xD040" />
-        <Animation Name="object_giant_Anim_00DE84" Offset="0xDE84" />
-        <Animation Name="object_giant_Anim_0102A4" Offset="0x102A4" />
-        <Animation Name="object_giant_Anim_0116E4" Offset="0x116E4" />
-        <Animation Name="object_giant_Anim_012A38" Offset="0x12A38" />
-        <Animation Name="object_giant_Anim_013004" Offset="0x13004" />
-        <Animation Name="object_giant_Anim_013FE8" Offset="0x13FE8" />
-        <Animation Name="object_giant_Anim_015334" Offset="0x15334" />
-        <Animation Name="object_giant_Anim_017944" Offset="0x17944" />
+        <!-- Giant Unused Animation -->
+        <Animation Name="gGiantLargeStrideAnim" Offset="0x2168" /> <!-- Original name is "kya_testwalk" -->
+
+        <!-- Giant Body Vertices. Contains everything but the beard. -->
+        <Array Name="gGiantBodyVtx" Count="444" Offset="0x2180">
+            <Vtx/>
+        </Array>
+
+        <!-- Giant Unused Beard Vertices. An exact duplicate of the beard vertices that are actually used. -->
+        <Array Name="gGiantUnusedBeardVtx" Count="9" Offset="0x3D40">
+            <Vtx/>
+        </Array>
```

Comment:
I'm not sure about only doing some of the Vtx... is there a reason to single this one out?

Reply:
Because it was this blob in the original XML:
```xml
        <!-- <Blob Name="object_giant_Blob_003D40" Size="0x90" Offset="0x3D40" /> -->
```
I didn't want to leave the blob as-is, so I defined it. But then having *only* that Vtx looked weird, so I defined the body and (used) beard Vtx too. But the body Vtx could be broken up further into individual limbs and pieces of limbs...not sure what the best solution for this is, because documenting every single set of Vtx sounds like a pain.

Reply:
If the body vtx are all in one big lump (and ZAPD has extracted it as such), I think something like `gGiantBodyVtx` would be fine. Or maybe `model`, but that would require discussion.

Reply:
Yeah, ZAPD extracted them as one big chunk; all I did is give them a name. Z64Utils breaks them down into 51 separate batches of vertices, but they're all adjacent, so I guess ZAPD either didn't notice or didn't care and just grabbed the entire thing as if it were one big chunk.

Reply:
This is to do with how they're used: most flex skeletons seem to put them in one big lump and access them as an array. Non-flex skeletons are rather worse.